tools/dragen-somatic/3.9.3/dragen-somatic__3.9.3.cwl

```diff
@@ -0,0 +1,962 @@
+cwlVersion: v1.1
+class: CommandLineTool
+
+# Extensions
+$namespaces:
+    s: https://schema.org/
+    ilmn-tes: http://platform.illumina.com/rdf/ica/
+$schemas:
+  - https://schema.org/version/latest/schemaorg-current-http.rdf
+
+# Metadata
+s:author:
+    class: s:Person
+    s:name: Sehrish Kanwal
+    s:email: sehrish.kanwal@umccr.org
+
+# ID/Docs
+id: dragen-somatic--3.9.3
+label: dragen-somatic v(3.9.3)
+doc: |
+    Documentation for dragen-somatic v3.9.3
```

Comment:
Might be worth putting some info in here, for 3.7.5 we have

>  Run tumor-normal dragen somatic pipeline v 3.7.5.
> Workflow takes in two separate lists of object stor version of the fastq_list.csv equivalent
> See the fastq_list_row schema definitions for more information.
> More information on the documentation can be found [here](https://sapac.support.illumina.com/content/dam/illumina-support/help/Illumina_DRAGEN_Bio_IT_Platform_v3_7_1000000141465/Content/SW/Informatics/Dragen/GPipelineSomCom_appDRAG.htm)

Reply:
Fixed in https://github.com/umccr/cwl-ica/pull/22/commits/30f53966ef9228f9dec8b9e55935ce6c06d17c09